mautic/mautic
# Template email detail view fixed, email line chart has the failed option

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Yes
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed issues  |  N/A

## Description
I found out while testing https://github.com/mautic/mautic/issues/1524 that the template email has different chart in the detail page than the list email. That chart did not work anymore because the charts were rewritten. I updated that chart to work with the new libraries.

Also, the "Emails in time" dashboard widget has 2 new options to display failed or sent, read and failed emails.

## Steps to reproduce the bug (if applicable)
Create a new template email or try to open the detail page of an existing one.

## Steps to test this PR
The detail page of the template email should display now, the chart has the date range filter. Check if the chart displays correct data and that the date range filter works.

Also, create a new "Emails in time" dashboard widget or edit an existing widget and test the new options.
